socioeconomic-differences/socioeconomic-differences.md

```diff
@@ -12,66 +12,88 @@ A person's socioeconomic status can inform their identity, as it affects the com
 
 ### Objectives
 **Participants will be able to:**
-- Anticipate certain pitfalls and aversions that come with belonging to a certain socioeconomic class
+    
+	  - Anticipate certain pitfalls and aversions that come with belonging to a certain socioeconomic class
+    - Learn the value of social capital and importance of asking for help from their fellow colleagues and seniors.
+    - Learn that it’s normal for white-collar professionals to want reasons behind tasks and therefore one must not panic but rather give an appropriate reason for the decision.
+
 
 ### Specific Things To Teach
-- The term "class straddler"
-- The value of social capital
-- The priority of networking
+   
+   - The term "class straddler"
+   - The value of social capital
+   - The priority of networking
 
 ### Materials
 
-- Limbo: Blue-Collar Roots, White-Collar Dreams by Alfred Lubrano
-- Hidden Rules of Class at Work by Ruby K. Payne and Don L. Krabill
-- Hillbilly Elegy by J.D. Vance
-- [Example video (10 min)](https://example.com) - Write a very short description of this resource.
-- [Other example article(20 min read)](https://otherexample.com) - Answer the 3 questions at the end.
-(Be sure to explain how to use these materials in directions in one of the sections below (ie: lesson, guided practice).  If you are not assigning them somehow, they should be placed under Supplemental Materials at the bottom.) 
-
-### Lesson
-
-Here's text about introducing something and how it works.
+    - Limbo: Blue-Collar Roots, White-Collar Dreams by Alfred Lubrano
```

Comment:
Either find a link for these resources, or remove them and put them in "supplemental resources".  They won't have time to read a book that day.

Reply:
Also, you should have the slideshow linked here.